src/style/common/slider.less

```diff
@@ -58,11 +58,11 @@
 
   &:checked + button,
   &:checked + label {
-    background-color: var(--green-500);
+    background-color: var(--accent-color-500);
```

Comment:
this one should stay green not accent one

Reply:
done